server/grpc_service.go

```diff
@@ -52,11 +57,8 @@ var (
 )
 
 // GetMembers implements gRPC PDServer.
-func (s *Server) GetMembers(context.Context, *pdpb.GetMembersRequest) (*pdpb.GetMembersResponse, error) {
-	if s.IsClosed() {
-		return nil, status.Errorf(codes.Unknown, "server not started")
-	}
-	members, err := cluster.GetMembers(s.GetClient())
+func (s *GrpcServer) GetMembers(context.Context, *pdpb.GetMembersRequest) (*pdpb.GetMembersResponse, error) {
```

Comment:
BTW, do we need to check the cluster ID?

Reply:
It is by design. The first request client sends to pd-server is `GetMembers` and the client does not know the cluster ID.

Reply:
I think I can add some comments here.